### PR TITLE
product and category sections are highlighted

### PR DESCRIPTION
**Pull request by AnastasiiaKolodiazhna**
Знімок екрана (191)
Знімок екрана (192)
Знімок екрана (193)
Знімок екрана (194)

### Checklist
- [ ] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅ All tests passed locally
- [ ] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [ ] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/categories/categories.js [hunk 1]
```diff
@@ -76,7 +76,9 @@ const Categories = () => {
           }
           name={category.name.length ? category.name[0].value : ''}
           deleteHandler={() => handleDeleteCategory(category._id)}
-          editHandler={() => dispatch(push(`/add-category/${category._id}`))}
+          editHandler={() =>
+            dispatch(push(`/categories/add-category/${category._id}`))
```
Comment (AndriyLymych) [inline src/pages/categories/categories.js:80]: dont use magic strings

File: src/pages/products/products-page.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ const ProductsPage = () => {
   };
 
   const handleProductEdit = (id) => {
-    dispatch(push(`/product/${id}`));
+    dispatch(push(`/products/product/${id}`));
```
Comment (AndriyLymych) [inline src/pages/products/products-page.js:94]: same

**Comment by AndriyLymych** [comment]
image
fix tests

**Comment by AndriyLymych** [comment]
image
link pr

**Review by AndriyLymych — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**